sdk/nextjs/src/server/setupDevCycle.tsx

```diff
@@ -54,7 +76,10 @@ export const setupDevCycle = <
             clientSDKKey,
             userGetter,
             options,
-        )
+        ).then((result) => {
```

Comment:
can this function be async?

Reply:
no, this is explicitly passing an unresolved promise